content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -397,71 +295,94 @@ The scheduler ensures that the sum of the resource requests of the scheduled Con
 
 ### How Pods with ephemeral-storage limits run
 
-For container-level isolation, if a Container's writable layer and logs usage exceeds its storage limit, the Pod will be evicted. For pod-level isolation, if the sum of the local ephemeral storage usage from all containers and also the Pod's emptyDir volumes exceeds the limit, the Pod will be evicted.
+For container-level isolation, if a Container's writable layer and logs
+usage exceeds its storage limit, the Pod will be evicted. For pod-level
+isolation, if the sum of the local ephemeral storage usage from all
+containers and also the Pod's emptyDir volumes exceeds the limit, the
+Pod will be evicted.
 
-### Monitoring ephemeral-storage consumption
+{{< caution >}}
+If the kubelet is not managing local ephemeral storage, then a container that uses more than its storage limit will not be evicted.
+{{< /caution >}}
 
-When local ephemeral storage is used, it is monitored on an ongoing
-basis by the kubelet.  The monitoring is performed by scanning each
-emptyDir volume, log directories, and writable layers on a periodic
-basis.  Starting with Kubernetes 1.15, emptyDir volumes (but not log
-directories or writable layers) may, at the cluster operator's option,
-be managed by use of [project
-quotas](http://xfs.org/docs/xfsdocs-xml-dev/XFS_User_Guide/tmp/en-US/html/xfs-quotas.html).
-Project quotas were originally implemented in XFS, and have more
-recently been ported to ext4fs.  Project quotas can be used for both
-monitoring and enforcement; as of Kubernetes 1.16, they are available
-as alpha functionality for monitoring only.
 
-Quotas are faster and more accurate than directory scanning.  When a
-directory is assigned to a project, all files created under a
-directory are created in that project, and the kernel merely has to
-keep track of how many blocks are in use by files in that project.  If
-a file is created and deleted, but with an open file descriptor, it
-continues to consume space.  This space will be tracked by the quota,
-but will not be seen by a directory scan.
+### Monitoring ephemeral-storage consumption {#resource-emphemeralstorage-consumption}
 
-Kubernetes uses project IDs starting from 1048576.  The IDs in use are
-registered in `/etc/projects` and `/etc/projid`.  If project IDs in
-this range are used for other purposes on the system, those project
-IDs must be registered in `/etc/projects` and `/etc/projid` to prevent
-Kubernetes from using them.
+If the kubelet is managing local ephemeral storage, then the kubelet
+measures storage use in:
+- emptyDir volumes
+- log directories
+- writeable container layers
 
-To enable use of project quotas, the cluster operator must do the
-following:
+This information can trigger Pod eviction if a Pod is using more
```

Comment:
```suggestion
This information triggers Pod eviction if a Pod is using more
```

Reply:
I think I'll try to rewrite it:
```markdown
If a Pod is using more ephemeral storage than you allow it to, the kubelet
sets an eviction signal that triggers Pod eviction.
```

but I need to get that checked for technical accuracy.